_topic_maps/_topic_map.yml

```diff
@@ -2229,6 +2229,14 @@ Topics:
   - Name: Placing pods on specific nodes using node selectors
     File: nodes-pods-node-selectors
     Distros: openshift-enterprise,openshift-origin
+  Name: Using run-once pods
+  Distros: openshift-enterprise
+    Dir: run_once_duration-override
+    Topics:
+    - Name: Run once duration override operator release notes
+      File: nodes-pods-run-once-duration-override-release-notes
```

Comment:
These file names should match the names in your nodes/pods/run_once_duration_override/ folder (minus the .adoc extension)